### PR TITLE
Add rhel10 scenario for daily runs

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: .github/workflows/scenarios-permian.yml [hunk 1]
```diff
@@ -133,6 +137,9 @@ jobs:
           elif [ "${{ matrix.scenario }}" == "rhel9" ]; then
             echo "installation_tree=http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.4.0/compose/BaseOS/x86_64/os" >> $GITHUB_OUTPUT
             echo "modular_url=http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.4.0/compose/AppStream/x86_64/os" >> $GITHUB_OUTPUT
+          elif [ "${{ matrix.scenario }}" == "rhel10" ]; then
+            echo "installation_tree=http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10/latest-RHEL-10.0.0/compose/BaseOS/x86_64/os" >> $GITHUB_OUTPUT
+            echo "modular_url=http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10/latest-RHEL-10.0.0/compose/AppStream/x86_64/os" >> $GITHUB_OUTPUT
```
Comment (jstodola) [inline .github/workflows/scenarios-permian.yml:142]: You should use these for RHEL-10 Public Beta testing:
`http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10.0/compose/BaseOS/x86_64/os/` and 
`http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10.0/compose/AppStream/x86_64/os/`

Comment (rvykydal) [reply .github/workflows/scenarios-permian.yml:142]: Good catch, thank you.

**Comment by rvykydal** [comment]
> Also the following line should be updated?
> 
> https://github.com/rhinstaller/kickstart-tests/blob/7f7c42a24cf4fc8f2d32704c4fc20932073049fb/scripts/weekly-summary#L46

For our workflows it should get artifacts as a parameter so I left it out, but it shouldn't hurt to add it also here.

**Comment by rvykydal** [comment]
Depends on https://github.com/rhinstaller/kickstart-tests/pull/1070

File: .github/workflows/scenarios-permian.yml (hunk 1 above)
Comment (rvykydal) [reply .github/workflows/scenarios-permian.yml]: Updated

**Review by jstodola — APPROVED** [review]
It looks good to me.

**Review by jkonecny12 — APPROVED** [review]
LGTM

**Pull request merged**